org-roam-refile.el

```diff
@@ -30,9 +30,70 @@
 ;; Org-roam refile allows you to refile notes to your nodes.
 ;;
 ;;; Code:
+(eval-when-compile
+  (require 'org-roam-macs)
+  (require 'org-macs))
+
 (defvar org-auto-align-tags)
 (defvar org-loop-over-headlines-in-active-region)
 
+(defun org-roam--file-keyword-get (keyword)
+  "Pull a keyword setting from the top of the file."
+  (nth 1
+     (assoc keyword
+            (org-collect-keywords (list keyword))))
+)
+
+(defun org-roam--file-keyword-kill (keyword)
+  "Erase a keyword setting line from the top of the file."
+  (let ((case-fold-search t))
+    (org-with-point-at 1
+      (when (re-search-forward (concat "^#\\+" keyword ":") nil t)
+        (beginning-of-line)
+        (delete-region (point) (line-end-position))
+        (delete-char 1))
```

Comment:
Use `kill-line` instead. See https://gist.github.com/jethrokuan/02f41028fb4a6f81787dc420fb99b6e4#file-migrate-el-L24

Reply:
I wanted to avoid putting the removed text in the kill ring, and `kill-line` doesn't have an option to avoid that. This ugly delete sequence was the best alternative I could find.